ts/features/itwallet/machine/credential/machine.ts

```diff
@@ -221,7 +222,7 @@ export const itwCredentialIssuanceMachine = setup({
       entry: "navigateToCredentialPreviewScreen",
       on: {
         "add-to-wallet": {
-          actions: ["storeCredential", "navigateToWallet"]
+          actions: ["storeCredential", "navigateToWallet", "trackAddCredential"]
```

Comment:
Why not directly use the `storeCredential` action to track the "Add credential" event?